Tests/NIOSSLTests/SSLCertificateTest.swift

```diff
@@ -480,12 +480,14 @@ class SSLCertificateTest: XCTestCase {
     }
 
     func testNotValidAfter() throws {
+        #if arch(arm64) || arch(x86_64)
```

Comment:
Thanks for this! It'd probably be better to express this constraint as "the system must have a 64-bit `time_t`", just for future-proofing. That would mean something like:

```swift
guard MemoryLayout<time_t>.size >= 8 else {
    throw XCTSkip()
}
```